lib/ansible/plugins/inventory/constructed.py

```diff
@@ -47,6 +48,9 @@
         # complex group membership
         multi_group: (group_names | intersect(['alpha', 'beta', 'omega'])) | length >= 2
 
+        # add a variable from environment lookup
+        gce_projects: "lookup('env', 'GCE_PROJECTS')"
```

Comment:
i would not use 'env' as example since they should use the config system 'env' keys to get environment variables